README.md

```diff
@@ -257,47 +260,43 @@ CKB> account import --ledger 0x69c46b6dd072a2693378ef4f5f35dcd82f826dc1fdcc89125
 
 Now that the account has been imported, it is remembered by the client and is visible when you run `account list`.
 
-### Get BIP44 Wallet Public Keys ###
-
-Use the `account bip44-addresses` command to obtain the BIP44 addresses for the mainnet.
+## Using your Account's Addresses ##
 
-Note that this command is provided as a convenience by the `ckb-cli` to get a list of addresses with the derivation path quickly.
+### Get BIP44 Address Public Keys ###
 
-**It is highly recommended to verify the account provided by this command on the Ledger device using the `account extended-address` command as described next.**
+Use the `account bip44-addresses` command to obtain the first 20 receiving and 10 change addresses for your account. 
 
 ``` sh
-CKB> account bip44-addresses --lock-arg 0x327a95bd57966e686ffe590c331cd37002e1c631
+CKB> account bip44-addresses --lock-arg <lock-arg>
 ```
 
+Note that this command is provided as a convenience by the `ckb-cli` to get a list of addresses with the derivation path quickly. Before sharing one of these receiving addresses, **it is highly recommended that you verify the address provided by this command on the Ledger device using the `account extended-address` command as described next.**
+
 ### Obtain / Verify Public Key ###
 
 The `account extended-address` command should be used to
 
 - Verify the public key obtained via `account bip44-addresses` command on the Ledger device
 - Obtain the public key for any arbitrary BIP44 derivation path
 
+Here's an example command that provide the first receiving address for the lock-arg `0x327a95bd57966e686ffe590c331cd37002e1c631`:
+
 ``` sh
 CKB> account extended-address --path "m/44'/309'/0'/0/1" --lock-arg 0x327a95bd57966e686ffe590c331cd37002e1c631
 ```
 
-This should show up on the ledger as (in 2 screens):
+This should show up on the ledger as 2 prompts:
 
-``` text
-Provide 
-Public Key
-```
-``` text
-Address:
-ckb1qyqxxtzygxvjwhgqklqlkedlqqwhp0rqjkvsqltkvh
-```
-If you've changed the Ledger's configuration to show testnet address, the last screen will instead look like this:
+|   Prompt 1   	|                     Prompt 2                     	|
+|:------------:	|:------------------------------------------------:	|
+|   `Provide`  	|                     `Address`                    	|
+| `Public Key` 	| `ckb1qyqxxtzygxvjwhgqklqlkedlqqwhp0rqjkvsqltkvh` 	|
 
-``` text
-Address:
-ckt1qyqxxtzygxvjwhgqklqlkedlqqwhp0rqjkvsa64fqt
-```
+**Verifying the output address printed by `ckb-cli` matches the one shown on Ledger prompt is highly recommended. Please read [Ledger's documentation](https://support.ledger.com/hc/en-us/articles/360006433934) on the subject.**
 
-After accepting the prompt on the Ledger the output on `ckb-cli` should look like:
+*Note: If you've changed the Ledger's configuration to show testnet address, the `Address` prompt will instead show the testnet address. This setting persists between power cycles.*
```

Comment:
*"if youve changed the nervos-app's configuration" maybe?

Reply:
Pushed a fix, how's this?